cmd/protoc-gen-goclay/internal/snake.go

```diff
@@ -0,0 +1,23 @@
+package internal
+
+import (
+	"unicode"
+)
+
+// ToSnake convert the given string to snake case following the Golang format:
+// acronyms are converted to lower-case and preceded by an underscore.
+// copied form https://gist.github.com/elwinar/14e1e897fdbe4d3432e1
+func ToSnake(in string) string {
```

Comment:
Let's copy-paste from https://github.com/azer/snakecase

It's a slightly improved version of the same gist.

I've already benchmarked 4 different libs here - https://github.com/mwf/goplay/tree/master/snakecase 
And it's the best an the simplest at the same time :)